test/MUXControls.Test/Infra/Application.cs

```diff
@@ -103,6 +108,9 @@ internal void Initialize(bool doLaunch = false, string deploymentDir = null)
                 }
                 else
                 {
+                    // Maxmize window to ensure we can find the corewindow
+                    ShowWindow(Hwnd, SW_SHOWMAXIMIZED);
```

Comment:
Is there a way to do this via UIA/MITA instead of using pinvoke? Using UIA or MITA here would be cleaner.